autopush-common/src/errors.rs

```diff
@@ -108,8 +108,6 @@ pub enum ApcErrorKind {
     ParseUrlError(#[from] url::ParseError),
     #[error(transparent)]
     ConfigError(#[from] config::ConfigError),
-    #[error(transparent)]
-    DbError(#[from] crate::db::error::DbError),
```

Comment:
`ApcError`'s gonna get significant cleanup when we kill the `autopush` crate, so since this was only used in one place I just killed it